doc/operation_guides/manual/installation-guide.md

```diff
@@ -219,16 +219,16 @@ prevents you from such errors.
 
 ```bash
 [openproject@host] cd ~/openproject
-[openproject@host] bundle exec rake db:create:all
-[openproject@host] bundle exec rake generate_secret_token
-[openproject@host] RAILS_ENV="production" bundle exec rake db:migrate
-[openproject@host] RAILS_ENV="production" bundle exec rake db:seed
-[openproject@host] RAILS_ENV="production" bundle exec rake assets:precompile
+[openproject@host] ./bin/rake db:create
+[openproject@host] ./bin/rake generate_secret_token
```

Comment:
We should set a proper environment variable `echo "SECRET_KEY_BASE=`bundle exec rake secret`" >> /etc/environment`